source/ed/osm2ed.cpp

```diff
@@ -86,7 +86,7 @@ void ReadRelationsVisitor::relation_callback(uint64_t osm_id,
                                                      << " is out of date: " << end_date << " (dropped)");
                     return;
                 }
-            } catch (boost::bad_lexical_cast& exception) {
+            } catch (...) {
```

Comment:
Okay, catch all exceptions

Reply:
As seen orally `const std::exception& e` with a `e.what()` in log (info or warn) might be better.

Reply:
I imagine that we want to catch only std::exceptions. Okay for me